### PR TITLE
5280/israel/h1 h2 changes

### PR DESCRIPTION
**Pull request by Israellund**
## Link to Issue
Closes: #5280 

## Description of Changes
- Changed h1 and h2 font sizes and styles

## "How We Fixed It"

-changed font sizes and added styles in react_quill_editor.scss and quill_formatted_text.scss, changed Title size in CommentCard to 28px as per Jess designs

NOTE: I was unable to figure out how to change the font size in the quill editor, despite being able to change the font family. What I think is happening is every time a new editor is opened the default settings from quill are overriding the css font size styles we have in place. The default font sizes can be referenced [here from the quill docs](https://quilljs.com/docs/formats/)(inspect their editor and see the default pixel sizes for h1 and h2, 26px and 19.5px respectively).

### CODE REVIEW

**Comment by masvelio** [comment]
you should simply target editors' h1 and h2, being specific enough (in terms of css specificity) to override `.ql-snow .ql-editor h1` and .`ql-snow .ql-editor h2` to proper values, so, respectively, 24px and 20px

File: packages/commonwealth/client/styles/components/quill/quill.snow.css [hunk 1]
```diff
@@ -557,7 +557,7 @@
 .ql-toolbar.ql-snow {
   border: 1px solid #ccc;
   box-sizing: border-box;
-  font-family: helvetica neue, helvetica, arial, sans-serif;
+  font-family: times, helvetica neue, helvetica, arial, sans-serif;
```
[the remark this answers is not in the document]
Comment (masvelio) [reply packages/commonwealth/client/styles/components/quill/quill.snow.css:560]: ?

File: packages/commonwealth/client/styles/components/react_quill/react_quill_editor.scss [hunk 2]
```diff
@@ -116,6 +116,7 @@
 
   .ql-editor {
     min-height: 212px;
+    font-family: $font-family-silka;
```
[the remark this answers is not in the document]
Comment (masvelio) [reply packages/commonwealth/client/styles/components/react_quill/react_quill_editor.scss:119]: okay so right now the whole text in editor is in Silka font but the thread and comment body is in Neue Haas Unica. Is this expected @jessmart1213  ?

https://github.com/hicommonwealth/commonwealth/assets/14819225/b05d49f1-2b94-477a-ab7f-79af5b23e2d8

**Review by mzparacha — COMMENTED** [review]
I have some similar questions as @masvelio asked above.

**Review by CowMuon — APPROVED** [review]
lgtm

**Review by rbennettcw — APPROVED** [review]

**Comment by jnaviask** [comment]
cc @Israellund to address Marcin's comments

**Pull request closed without merging**